easybuild/easyconfigs/i/icc/README.md

```diff
@@ -0,0 +1,13 @@
+In order to use these configs you must first download the Intel Parallel Studio XE CPP file. NOT the full bundle! The file should look like this: parallel_studio_xe_2018_update2_composer_edition_for_cpp.tgz    
```

Comment:
Actually, you can also use the full bundle as a source file, but then you need to change the `sources` spec in the easyconfig file accordingly...

We list which `components` that should be installed exactly for this reason: to ensure that only `icc` installed even if you use the full bundle to install...

Reply:
I didn't know that. But for someone reading this documentation trying to get it working the first time, do we want to complicate things by asking them to modify code first? Or use a simple method to first get it working? Personally, I prefer to just get something that works then explore the other options.

Reply:
Well, sure, simple approach is better, but we shouldn't claim it's not possible, since it is (with minimal effort).

Reply:
Good point. I will need some time to prove this out and write up the documentation.

It generates a question though, if a full bundle is supported and if there is a primary [Intel script](https://github.com/easybuilders/easybuild-easyconfigs/tree/master/easybuild/easyconfigs/i/intel) then wouldn't it be better to have the documentation for installing the bundle be in the primary Intel section (that way someone gets ALL of the Intel tools at once) then have the documentation for the individual applications be specific to them?

This question may be a bit idealistic in nature considering that the primary intel scripts don't auto include things like the cluster checker, vtune, ect. But maybe it should?

Reply:
@boegel I had to do heavy modifications to get the full bundle to install. It didn't like any of the patches, had to change the source, and the sha256sum.

Would it not be better to create a whole new file for installing from the full bundle?

Reply:
I think we currently assume in a couple of places that the `intel` toolchain components like `icc`, `ifort`, `impi` and `imkl` are installed separate (or at least that the corresponding `$EBROOT*` environment variables are set), so it's probably less trivial than it seems...

It's not entirely clear to me what the best approach is here. When do people have access to (only) the full bundle, when do they have access to separate tarballs for each component/tool?

Reply:
So it depends on which version of the tools you buy (see #6346 for the work I've done in trying to make sense of it). Each of those is a bundle of tools and I honestly don't know why anyone would want to install just individual components on a system*. I don't think you can buy just an individual tool anymore; so most of the time you have a bundle of at least a compiler, MKL, DAAL, IPP, and ITB. The bigger bundles have more tools and Cluster has them all.

* The only time I've ever just installed a single product on the cluster outside of a bundle is when I got bit bad by a bug and they released a point release which was literally one package update in one tool...so I just updated that one package.

With that said, when you have access to a bundle you have access to the individual products as well. I'm liking the idea of a "master" that installs the bundle by calling the individual projects. I started hacking away on that but life got busy and I've been swamped at work recently. *shrug* it goes that way sometimes. I will get back to hacking on EB soon. :-)

Thanks!